docs/dev/new_check_howto.md

```diff
@@ -300,9 +300,59 @@ The `logos` folder must contain **three** images with filenames and sizes that m
+| `creates_events`    | Boolean         | Mandatory          | If the integration should be able to create events. If this is set to `false`, attempting to create an event from the integration results in an error.                                                                             |
+| `display_name`      | String          | Mandatory          | Title displayed on the corresponding integration tile in the Datadog application and on the[Public documentation Integrations page][12]                                                                                            |
+| `doc_link`          | String          | Mandatory          | Link to the documentation page of the integration. Format is `https://docs.datadoghq.com/integrations/<INTEGRATION_NAME>`                                                                                                          |
+| `guid`              | String          | Mandatory          | Unique ID for the integration. [Generate a UUID][13]                                                                                                                                                                               |
+| `is_public`         | Boolean         | Mandatory          | If set to `false` the integration `README.md` content is not indexed by bots in the Datadog public documentation.                                                                                                                  |
+| `maintainer`        | String          | Mandatory          | Email of the owner of the integration.                                                                                                                                                                                             |
+| `manifest_version`  | String          | Mandatory          |                                                                                                                                                                                                                                    |
+| `name`              | String          | Mandatory          | Unique name for the integration. Use the folder name for this parameter.                                                                                                                                                           |
+| `public_title`      | String          | Mandatory          | Title of the integration displayed on the documentation. Should follow the following format: `Datadog-<INTEGRATION_NAME> Integration`                                                                                              |
+| `short_description` | String          | Mandatory          | This text -Maximum 80 characters- appears at the top of the integration tile as well as the integration's rollover text on the integrations page.                                                                                  |
```

Comment:
Should we add a check as well for the number of chars in the description

Reply:
🤔 that would be nice to have for the short_description. The description can technically be as long as we want.